src/traces/parcoords/parcoords.js

```diff
@@ -24,6 +24,17 @@ var c = require('./constants');
 var brush = require('./axisbrush');
 var lineLayerMaker = require('./lines');
 
+var tickText = require('../../plots/cartesian/axes').tickText;
+var parcoordsLayout;
+
+function linearFormat(v) {
+    return tickText(
+        parcoordsLayout.linearAxis,
+        parcoordsLayout.linearAxis.d2l(v),
+        true
```

Comment:
Why are you setting the `hover:true` flag here?

https://github.com/plotly/plotly.js/blob/06901623e988b7ad036207a905a4a032da186357/src/plots/cartesian/axes.js#L934-L940

Does it make the formatting look more like what's we see on master?

Reply:
`hover: false` works after https://github.com/plotly/plotly.js/pull/3966/commits/9e73c277ab5e8fee4df3056efa791b4ee292a067.